db/a3a

```diff
@@ -1,6 +1,6 @@
 R2PM_BEGIN
 
-R2PM_GIT "https://github.com/radare/radare2-rust"
+R2PM_GIT "https://github.com/radareorg/radare2-rust"
```

Comment:
This should be in extras

Reply:
nvm, its an archived repository. kill it with fire (its deleted in master)